src/js/pensions/components/GetFormHelp.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function GetFormHelp() {
```

Comment:
I think we could use the same component for pension and burials; it's the same call center for both.